tensorflow/python/ops/numpy_ops/np_array_ops.py

```diff
@@ -560,6 +560,19 @@ def _reduce(tf_fn,
       tf_fn(input_tensor=a.data, axis=axis, keepdims=keepdims))
 
 
+@utils.np_doc('size')
+def size(x):
```

Comment:
Looking at the documentation for np.size, it seems to have an axis argument as well. Can you add that as well? (We don't have to support the code for now, we can simply raise an error if the 'axis' argument is something other None)

Please add a test for that as well.

Reply:
Hi, Akshay! Should I also include tests for raising errors?

Reply:
Never mind. I included the according tests.